src/sidebar/components/moderation-banner.js

```diff
@@ -51,15 +51,16 @@ function ModerationBanner({ annotation, api, toastMessenger }) {
   };
 
   const toggleButtonProps = (() => {
```

Comment:
This is toggle button but we don't have an `aria-pressed` prop.  I suppose thats okay when we change the title like this. But I see we're not changing the title when using `aria-expanded` (in AnnotationBody) Should we also apply that standard for aria-pressed? I'm a bit confused by that.

Reply:
Can we please put non-label/title things aside for this set of changes?